src/js/store.js

```diff
@@ -1,12 +1,10 @@
 import { applyMiddleware, createStore, compose } from "redux"
-
 import thunk from "redux-thunk"
 import promise from "redux-promise-middleware"
-
 import reducer from "./reducers/index"
```

Comment:
If you import from `index.js` file You could write simply just name of folder: `import reducer from "./reducers"`